lib/ruby_lsp/listeners/hover.rb

```diff
@@ -123,7 +123,7 @@ def generate_gem_hover(node)
         # Remove leading whitespace if a heredoc was used for the summary or description
         info = info.gsub(/^ +/, "")
 
-        @response_builder.push("**#{spec.name}** (#{spec.version})", :title)
+        @response_builder.push("**#{spec.name}** (#{spec.version})", :code)
```

Comment:
Here's a good example of how our previous decision doesn't work. The gem's name and version isn't really code. And putting it inside a code block messes up the markdown styling.

Sorry for the trouble, but I think we need to revert this into `title` and not add the code block every time.

Reply:
I see your point.

I can undo my latest commit (to keep the 2 categories separate). Or are you suggesting something else?